examples/complete/variables.tf

```diff
@@ -5,3 +5,7 @@ variable "region" {
 variable "availability_zones" {
   type = list(string)
 }
+
+variable "security_group_suffix" {
+  type = string
```

Comment:
add description

Reply:
This is an example var and the other vars don't have one. If I add it for this one, I would add it for the others, no ?

Reply:
Done for all example vars